oraclebmc/core/models/attach_i_scsi_volume_details.py

```diff
@@ -2,11 +2,12 @@
 # Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
 
 from .attach_volume_details import AttachVolumeDetails
-from ...util import formatted_flat_dict
+from ...util import formatted_flat_dict, initkwargs
```

Comment:
You don't need to grab it in this review, but with the class decorator below we can move the use of `formatted_flat_dict` into the decorator as well; every model simply uses it for the same `__repr__`.